scripts/utils/trading_strategy_helpers.js

```diff
@@ -47,33 +48,46 @@ const maxUINT = new BN(2).pow(new BN(256)).sub(new BN(1))
  * {
  *   amount: 100,
  *   tokenAddress: 0x0000000000000000000000000000000000000000,
- *   userAddress: 0x0000000000000000000000000000000000000001
+ *   bracketAddress: 0x0000000000000000000000000000000000000001
  * }
  * @typedef Deposit
  * @type {object}
  * @property {integer} amount integer denoting amount to be deposited
- * @property {EthereumAddress} tokenAddress {@link EthereumAddress} of token to be deposited
- * @property {EthereumAddress} userAddress address of user depositing
+ * @property {Address} tokenAddress {@link Address} of token to be deposited
+ * @property {Address} bracketAddress address of bracket into which to deposit
  */
 
 /**
  * @typedef Withdrawal
  *  * Example:
  * {
  *   amount: "100",
- *   userAddress: "0x0000000000000000000000000000000000000000",
+ *   bracketAddress: "0x0000000000000000000000000000000000000000",
  *   tokenAddress: "0x0000000000000000000000000000000000000000",
  * }
  * @type {object}
  * @property {integer} amount Integer denoting amount to be deposited
- * @property {EthereumAddress} userAddress Ethereum address of the trader performing the withdrawal
- * @property {EthereumAddress} tokenAddresses List of tokens that the traded wishes to withdraw
+ * @property {Address} bracketAddress Ethereum address of the bracket from which to withdraw
+ * @property {Address} tokenAddresses List of tokens that the traded wishes to withdraw
  */
 
 const formatAmount = function(amount, token) {
   return new BN(10).pow(new BN(token.decimals)).muln(amount)
 }
 
+/**
+ * Checks that the first input address is the only owner of the first input address
+ * @param {Address} masterAddress address that should be the only owner
+ * @param {Address} ownedAddress address that is owned
+ * @return {bool} whether ownedAddress is indeed owned only by masterAddress
+ */
+const isOnlyOwner = async function (masterAddress, ownedAddress, artifacts) {
```

Comment:
Since this is so specific to the safe, isOnlySaveOwner might be a little bit better name.